test/node/node_test.cpp

```diff
@@ -155,19 +163,21 @@ TEST(NodeTest, SimpleSubkeys) {
 }
 
 TEST(NodeTest, StdArray) {
-  std::array<int, 5> evens;
-  evens[0] = 2;
-  evens[1] = 4;
-  evens[2] = 6;
-  evens[3] = 8;
-  evens[4] = 10;
-
+  std::array<int, 5> evens {{ 2, 4, 6, 8, 10 }};
   Node node;
   node["evens"] = evens;
-  std::array<int, 5> actualEvens = node["evens"].as<std::array<int, 5> >();
+  std::array<int, 5> actualEvens = node["evens"].as<std::array<int, 5>>();
   EXPECT_EQ(evens, actualEvens);
 }
 
+TEST(NodeTest, StdArrayWrongSize) {
+  std::array<int, 3> evens {{ 2, 4, 6 }};
+  Node node;
+  node["evens"] = evens;
+  EXPECT_THROW_REPRESENTATION_EXCEPTION((node["evens"].as<std::array<int, 5>>()),
```

Comment:
Would it be even better to inline and create 2 tests in order to have only 1 assertion per test? I think it would also make the tests less big and thus easier to read.

    TEST(NodeTest, StdArrayWrongSizeThrowsRightException) {
      ...
    TEST(NodeTest, StdArrayWrongSizeThrowsExceptionWithRightMessage) {
      ...

I personally don't find the camelcased test names very readable but I wrote it like that to stay consistant with the rest of the test suite :-/